forge.config.ts

```diff
@@ -86,7 +86,7 @@ const noopAfterCopy = (
 
 const config: ForgeConfig = {
   packagerConfig: {
-    appBundleId: 'com.bytedance.uitars',
+    // appBundleId: 'com.bytedance.uitars',
```

Comment:
why comment it

Reply:
After enabling, Mac requires the user to input the local password